x-pack/auditbeat/module/system/socket/helper/loopback.go

```diff
@@ -106,6 +108,7 @@ func (lo *IPv6Loopback) AddRandomAddress() (addr net.IP, err error) {
 		if err = unix.Bind(fd, &bindAddr); err == nil {
 			break
 		}
+		//nolint:errorlint // ignore
 		if errno, ok := err.(unix.Errno); !ok || errno != unix.EADDRNOTAVAIL {
```

Comment:
This should be expressible as `if err != unix.EADDRNOTAVAIL {`; `syscall.Errno` is an `error` so they are comparable. Caveat if someone ever implements an error that is not on a pointer and contains a slice or a map then ¡bang!, but that's true for most uses of errors.